webcompat/static/js/lib/models/issue.js

```diff
@@ -83,7 +72,7 @@ issues.Issue = Backbone.Model.extend({
       state: response.state,
       title: this.getTitle(
         this.getDomain(response.title),
-        this.getDescription(response.body),
+        this.getDescription(response.body_html),
```

Comment:
This doesn't seem necessary anymore. It was necessary because we wanted the raw version and not the `md.render` one.

`this.getDescription(body)` should work.

Reply:
Sorry, I didn't understand what you mean.
We need the `body_html` because is the name of the attribute. 
I tried to replace for `this.getDescription(body)` but it didn't work.

Reply:
yup. I'm just saying it's already defined just above in the code.

https://github.com/webcompat/webcompat.com/blob/08336441c3c00a2c3189f629e666f4b8ab8b9a6c/webcompat/static/js/lib/models/issue.js#L60-L78

or to make it more compact.

```js
    this.set({
      body: response.body_html,

// removed lines for readability

      title: this.getTitle(
        this.getDomain(response.title),
// body has been assigned above already.
        this.getDescription(body),
        response.title
      )
});
```

That said I'm not a JS expert, and maybe it's not good to do this :)